### PR TITLE
TrilinosCouplings: Minor Improvements to the Poisson STK example

### PR DESCRIPTION
**Pull request by csiefer2**
Now supporting more than one nodeset for BCs.

### CODE REVIEW

File: packages/trilinoscouplings/examples/scaling/example_Poisson_stk.cpp [hunk 1]
```diff
@@ -646,15 +654,36 @@ int main(int argc, char *argv[]) {
...
     }      
   } // end loop over elements
   if(coordsFilename != "") {
+    // WARNING: This does *NOT* output the coordinates in an ordering that corresponds
+    // to the matrix unless the matrix is (a) in serial and (b) has identical global and ordering
+    // The ifdef'd code below will do processor-by-processor output with global IDs for matching
+    // in all other cases
     EpetraExt::MultiVectorToMatrixMarketFile(coordsFilename.c_str(),nCoord,0,0,false);
+
+#if 0
```
Comment (GrahamBenHarper) [inline packages/trilinoscouplings/examples/scaling/example_Poisson_stk.cpp:673]: :thinking:

**Review by GrahamBenHarper — DISMISSED** [review]
yep

File: packages/trilinoscouplings/examples/scaling/example_Poisson_stk.cpp (hunk 1 above)
Comment (csiefer2) [reply packages/trilinoscouplings/examples/scaling/example_Poisson_stk.cpp:673]: That's in there on purpose

Comment (csiefer2) [reply packages/trilinoscouplings/examples/scaling/example_Poisson_stk.cpp:673]: Read the comment above :)

**Pull request merged**